server/fleet/app.go

```diff
@@ -147,6 +151,20 @@ type MDM struct {
 	MacOSMigration        MacOSMigration           `json:"macos_migration"`
 	EndUserAuthentication MDMEndUserAuthentication `json:"end_user_authentication"`
 
+	// WindowsEnabledAndConfigured indicates if Fleet MDM is enabled for Windows.
+	// There is no other configuration required for Windows other than enabling
+	// the support, but it is still called "EnabledAndConfigured" for consistency
+	// with the similarly named macOS-specific fields.
+	WindowsEnabledAndConfigured bool `json:"windows_enabled_and_configured"`
+
+	// WindowsExcludedTeams is the list of team names that are excluded from the
+	// Windows MDM enrollment. Hosts that have Fleet Desktop installed and that
+	// are NOT part of those teams will be enrolled. Note that the setting stores
+	// team names instead of IDs because it may be set via 'fleetctl apply' and a
+	// YAML file, where names are used to identify entities (and this is
+	// consistent with the AppleBMDefaultTeam setting).
+	WindowsExcludedTeams []string `json:"windows_excluded_teams"`
```

Comment:
Regarding https://github.com/fleetdm/fleet/issues/12259#issuecomment-1591588979, is it that we _definitely_ don't want that feature to exclude by teams, or just we scoped it down for this release? Because if it's the latter, I'd suggest we leave it in there but only available via `fleetctl apply` (or the corresponding PATCH /config API). The UI doesn't have to support it immediately (though it wouldn't be able to display it as information either). 

Another option (without any impact on the UI this time) is to leave the code that supports it there, but add a validation that fails updating the settings if a team is excluded.

Reply:
cc: @zhumo

Reply:
FYI - [This](https://github.com/fleetdm/fleet/issues/11952#issuecomment-1591450900) is the gh comment where Mo mentioned that teams exclusion is no longer required. @zhumo Do you see the teams exclusions feature coming back in the future?

Reply:
@roperzh @marcosd4h Sounds good, seems like we definitely don't want them from the comment? In any case I'll go ahead and remove team exclusion altogether from the PR, it's easy enough to re-add if needed (from the code here: https://github.com/fleetdm/fleet/pull/12340/files/9066ed8303738b2218ad1d3ca548075a8901c6b8).

Reply:
Yeah instead of teams exclusion, we just want the agent to recognize whether it's on a windows server and not do the enrollment if it's on a Windows server